modules/get-started/pages/whats-new.adoc

```diff
@@ -21,6 +21,13 @@ You can now deploy data transform functions that xref:develop:data-transforms/de
 
 The docs now also include an xref:develop:data-transforms/index.adoc[expanded guide] designed to help you master the creation, deployment, and management of data transforms in Redpanda.
 
+
+== Client throughput management
+
+Redpanda now includes `rpk` and Redpanda Console support for managing xref:manage:cluster-maintenance/manage-throughput.adoc#client-throughput-limits[client throughput limits] with the AlterClientQuotas and DescribeClientQuotas Kafka APIs. You can set client throughput quotas using `rpk` for individual client based on client ID, as well as a group of clients based on ID prefix matching. New xref:manage:cluster-maintenance/manage-throughput.adoc#monitor-client-throughput[metrics] are available for insight on client throughput utilization.
+
+Client throughput quotas, previously applied on a per-shard basis, are also adjusted to apply on a per-broker basis. Cluster configuration properties for managing client quotas are deprecated, including `target_quota_byte_rate` which is disabled by default with the value `0`.
```

Comment:
```suggestion
Client throughput quotas, previously applied on a per-shard basis, are now apply on a per-broker basis. Cluster configuration properties for managing client quotas are deprecated, including `target_quota_byte_rate` which is disabled by default with the value `0`.
```

Reply:
Can you link to the deprecation entry in https://deploy-preview-592--redpanda-docs-preview.netlify.app/24.2/upgrade/deprecated/ ? Wondering why it's not listed here? If it is deprecated, then we should always list in this table (along with all other deprecated properties).